perfrunbook/utilities/measure_aggregated_pmu_stats.py

```diff
@@ -505,9 +505,25 @@ def get_counters(self) -> list:
         ArmCounterPKI("inst-tlb-tw-pki", "inst_tlb_walk", "event=0x35"),
         ArmCounterPKI("data-tlb-mpki", "data_tlb_refill", "event=0x5"),
         ArmCounterPKI("data-tlb-tw-pki", "data_tlb_walk", "event=0x34"),
+        ArmCounterPKC("flop-scalar-pkc", "VFP_SPEC", "event=0x75"),
+        # ArmCounterPKC("neon-pkc", "ASE_SPEC", "event=0x74"),
```

Comment:
Delete ASE_SPEC instead of leaving it commented.

Reply:
I have a hard time understanding what "ASE_SPEC" vs "ASE_INST_SPEC" is measuring, which is one reason I commented it out.  The PMU documentation isn't clear to me, but Geoff is right: ASE_SPEC is defined for Graviton 2, while ASE_INST_SPEC is introduced in Graviton 3.